packages/react-components/src/atoms/Link.tsx

```diff
@@ -6,7 +6,35 @@ import { fern, paper, pine } from '../colors';
 import { useHasRouter } from '../hooks';
 import { ThemeVariant, defaultThemeVariant } from '../theme';
 
-const styles = css({
+interface BaseLinkProps {
+  readonly href: string;
+  readonly styles: any;
+}
+
+const BaseLink: React.FC<BaseLinkProps> = ({ children, href, styles }) => {
+  if (useHasRouter()) {
+    return (
+      <ReactRouterLink css={styles} to={href}>
+        {children}
+      </ReactRouterLink>
+    );
+  }
+
+  const internal =
```

Comment:
I think this is the problem for the bug mentioned in standup. I think we are using the react router link because there is a router. We might need to test for both

Reply:
Hmm yeah. Tests should have caught that - I have an idea how to improve the tests so that they catch this kind of thing, will pick the bug up at some point. Thanks!